mythril/support/loader.py

```diff
@@ -47,6 +47,22 @@ def read_storage(self, contract_address: str, index: int) -> str:
             contract_address, position=index, block="latest"
         )
 
+    @functools.lru_cache(LRU_CACHE_SIZE)
+    def read_balance(self, address: str) -> str:
+        """
+
+        :param address:
+        :return:
+        """
+        if not self.storage_loading:
```

Comment:
we should change this variable to `onchain_loading` as this isn't storage right?

Reply:
I simplified this to a single boolean (see updated PR)